lib/pure/unittest.nim

```diff
@@ -185,7 +192,15 @@ proc defaultConsoleFormatter*(): ConsoleOutputFormatter =
     # Reading settings
     # On a terminal this branch is executed
     var envOutLvl = os.getEnv("NIMTEST_OUTPUT_LVL").string
-    var colorOutput  = not existsEnv("NIMTEST_NO_COLOR")
+    var colorOutput  = isatty(stdout)
+    if existsEnv("NIMTEST_COLOR"):
+      let colorEnv = getenv("NIMTEST_COLOR")
+      if colorEnv == "never":
+        colorOutput = false
+      elif colorEnv == "always":
+        colorOutput = true
+    elif existsEnv("NIMTEST_NO_COLOR"):
```

Comment:
This shouldn't be an 'elif'.

Reply:
Because ``NIMTEST_COLOR`` should override ``NIMTEST_NO_COLOR``, only if ``NIMTEST_COLOR`` is not defined in the env, we consider ``NIMTEST_NO_COLOR``.